perllib/Open311.pm

```diff
@@ -420,9 +420,12 @@ sub _get {
     $params->{ jurisdiction_id } = $self->jurisdiction
         if $self->jurisdiction;
     $uri->path( $uri->path . $path );
+    my $base_uri = $uri->clone;
     $uri->query_form( $params );
 
-    $self->debug_details( $self->debug_details . "\nrequest:" . $uri->as_string );
+    my $debug_request = "GET " . $base_uri->as_string . "\n";
+    $debug_request .= join("\n", map { "$_: $params->{$_}" } keys %$params);
```

Comment:
Maybe an extra \n in here just to split up the params from the request details for readability?